.github/workflows/build-test-e2e.yml

```diff
@@ -37,8 +37,7 @@ jobs:
         run: yarn playwright install --with-deps chromium
       - name: Run Playwright tests
         env:
-          ADMIN_USERNAME: ${{ secrets.ADMIN_USERNAME }}
-          ADMIN_PASSWORD: ${{ secrets.ADMIN_PASSWORD }}
+          ADMIN_PAT: ${{ secrets.ADMIN_PAT }}
```

Comment:
We'll need to add the `ADMIN_PAT` secret to the Sage SWC repo before merging these changes to develop. We can also remove the `ADMIN_USERNAME` and `ADMIN_PASSWORD` secrets.